### PR TITLE
Update tooling for OCP 4.15; update base images

### PR DESCRIPTION
**Pull request by AObuchow**
This PR updates the tooling used in the web terminal operator tooling container to their latest versions available on mirror.openshift.com and github releases for OCP 4.15, as well as updates the UBI8-minimal image used. 

# Testing
I've pushed the web terminal tooling container built from this PR to `quay.io/aobuchow/wto-tooling:1.10` for convenience. 

1. Launch an OpenShift 4.15 cluster
2. Install the web-terminal operator
3. Create a web terminal and click the link to specify a custom image, and provide `quay.io/aobuchow/wto-tooling:1.10`. Alternatively, create a web terminal with the default image and run `wtoctl set image quay.io/aobuchow/wto-tooling:1.10` in the web terminal console
4. Verify the web terminal starts up successfully. The provided binaries should be updated to the versions specified in this PR. For example, running `oc version` should indicate version 4.15.

### CODE REVIEW

**Comment by AObuchow** [comment]
I had some doubts w.r.t the version of kustomize that should be included in this PR, as the kustomize CLI binary is not included downstream (neither is `rhoas`). 

Downstream, we only ship kustomize as a kubectl plugin (i.e. `kubectl kustomize ...`). The version of kustomize can be checked with `kubectl version`. With oc 4.15 & kubectl v.1.28.2, the version of kubectl kustomize included is `v5.0.4-0.20230601165947-6ce0bf390ce3`. However, WTO 1.9 upstream was already shipping  v5.2.1 of the kustomize binary.

Thus, I'm under the impression the version of the kustomize binary we ship upstream is not dependant on the version of kubectl kustomize that we ship downstream.

File: README.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ Default OpenShift Console Web Terminal tooling container.
...
 - [odo](https://github.com/openshift/odo) [v3.15.0](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/odo/v3.15.0)
-- [tekton](https://github.com/tektoncd/cli) [v1.13.0](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/pipelines/1.13.0)
-- [knative](https://github.com/knative/client) [v1.9.2](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/serverless/1.9.2)
+- [tekton](https://github.com/tektoncd/cli) [v1.14.0](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/pipelines/1.14.0)
+- [knative](https://github.com/knative/client) [v1.10.0](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/serverless/1.10.0-6)
 - [rhoas](https://github.com/redhat-developer/app-services-cli) [v0.53.0](https://github.com/redhat-developer/app-services-cli/tree/v0.53.0)
-- [submariner](https://github.com/submariner-io/submariner) [v0.16.2](https://github.com/submariner-io/subctl/tree/v0.16.2)
+- [submariner](https://github.com/submariner-io/submariner) [v0.17.0](https://github.com/submariner-io/subctl/tree/v0.17.0)
 - [kubevirt](https://github.com/kubevirt/kubevirt) [v1.1.0](https://github.com/kubevirt/kubevirt/tree/v1.1.0)
```
Comment (dkwon17) [inline README.md:16]: Double checking, should kubevirt be updated as well? To [kubevirt 1.2.0](https://github.com/kubevirt/kubevirt/releases/tag/v1.2.0).

Comment (AObuchow) [reply README.md:16]: Good catch, seems like kubevirt was released as I was working on this PR

**Comment by dkwon17** [comment]
The output LGTM:
Screenshot from 2024-03-13 15-33-10

File: README.md (hunk 1 above)
Comment (AObuchow) [reply README.md:16]: Just double checked: WTO downstream uses the `cnv-4.14-for-rhel-8-x86_64-rpms` repo for installing kubevirt (virtctl) and there doesn't seem to be a correspondance between the downstream version and upstream version: the version shipped in both upstream and dowstream are the latest available versions for each product. 

For example, WTO 1.9 upstream uses kubevirt `v1.10` and downstream uses `v0.59.2-50-gfc77afd4e`.

Will update this PR accordingly.

**Comment by AObuchow** [comment]
Thank you for the review David :) I pushed the fixup commits for updating kubevirt to v1.2.0

**Review by dkwon17 — APPROVED** [review]
image

LGTM

**Comment by AObuchow** [comment]
Thank you so much for the review @dkwon17 :) Merging :tada:

**Pull request merged**